x/concentrated-liquidity/keeper_test.go

```diff
@@ -39,6 +40,13 @@ func (suite *KeeperTestSuite) SetupTest() {
 	suite.Setup()
 }
 
+// SetupDefaultPool sets up a eth usdc concentrated liquid pool with pool ID 1 and no liquidity
+func (s *KeeperTestSuite) SetupDefaultPool(ctx sdk.Context) types.ConcentratedPoolExtension {
```

Comment:
Because most of the existing pool creation helpers are using this naming convention.

```suggestion
func (s *KeeperTestSuite) PrepareDefaultPool(ctx sdk.Context) types.ConcentratedPoolExtension {
```